app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+    results = {}
+
+    def inner(*args):
+        if args in results is not None:
+            print("Getting from cache")
+            return results[args]
+
+        print("Calculating new result")
+        results[args] = func(*args)
+        return results[args]
```

Comment:
try to write your decorator in such way that the return statement is not duplicated